core/src/main/java/de/bwaldvogel/mongo/ServerVersion.java

```diff
@@ -5,7 +5,7 @@
 
 import de.bwaldvogel.mongo.backend.Utils;
 
-public enum ServerVersion {
+public enum ServerVersion implements MongoVersion {
```

Comment:
The `toVersionString()` method in the `ServerVersion` enum should be removed now since it’s redundant.